features/support/object_model_pages/ivl/registration/ivl_choose_plan.rb

```diff
@@ -90,4 +90,16 @@ def self.continue_btn
   def self.non_silver_plan_modal_text
     'You are Eligible for Lower Costs'
   end
+
+  def self.compare_checkbox
+    'span.checkbox-custom-label'
+  end
+
+  def self.compare_plans_btn
+    '.ivl-compare-selected-plans-link'
```

Comment:
let's use a `data-cuke="something"` selector for all new page object model selectors